v1/src/main/java/com/google/cloud/teleport/spanner/common/SizedType.java

```diff
@@ -196,6 +234,15 @@ public static SizedType parseSpannerType(String spannerType, Dialect dialect) {
         }
       case POSTGRESQL:
         {
+          if (spannerType.contains("[] vector length")) {
```

Comment:
Isn't `VECTOR LENGTH` also acceptable? Ignore if `spannerType` is guaranteed to be lowercase.

Reply:
well. This file doesn't treats spannertype as case sensitive. spannertype is reported by Spanner's Information Schema. 
however, I have made the regex case insensitive to make the code more robust.